src/lang/proc_macros/client/mod.rs

```diff
@@ -45,18 +45,22 @@ impl ProcMacroClient {
         }
     }
 
+    #[tracing::instrument(level = "trace", skip_all)]
```

Comment:
can you please post a screenshot of perfetto with new profiles now? how large are these files now?

Reply:
![image](https://github.com/user-attachments/assets/1a5296c5-70a1-44f7-b435-e2778ad246c4)
`cairo-ls:diagnostics-controller 2` green ones are `generate_code` calls and purple ones are mostly `request_attribute` calls.
All ops on the main thread are up to 3-4ms.
I ran on https://github.com/kkrt-labs/kakarot-ssj for 70 sec (all resolutions done, but please note it is a `cairo-test` version) file size of 14 Mb.